RevenueCat.xcodeproj/project.pbxproj

```diff
@@ -7125,7 +7125,7 @@
 				MTL_ENABLE_DEBUG_INFO = INCLUDE_SOURCE;
 				MTL_FAST_MATH = YES;
 				OTHER_SWIFT_FLAGS = "";
-				PRODUCT_BUNDLE_IDENTIFIER = com.revenuecat.RevenueCatUIDev;
+				PRODUCT_BUNDLE_IDENTIFIER = com.revenuecat.RevenueCatUI;
```

Comment:
The stuff in this file was preventing CI from archiving the XCFramework correctly, but it did work on my machine because I had the profiles installed for it and it was set to auto manage. 

The setup in this PR matches the one we've had forever for the `RevenueCat.xcframework`